pep-0639.rst

```diff
@@ -1,543 +1,2322 @@
+`SciPy <#scipyissue_>`_ also had similar issues, as reported by the
+original author of this PEP and now resolved for those cases.
+
+However, given the inherent complexity here and a lack of an obvious
+mechanism to do so, the fact that each wheel would need its own license
+information, lack of support on PyPI for exposing license info on a
+per-distribution archive basis, and the relatively niche use case, it was
+determined to be out of scope for this PEP, and left to a future PEP
+to resolve if sufficient need and interest exists and an appropriate
+mechanism can be found.
+
+
+Open Issues
+===========
+
+Should the ``License`` field be back-filled, or mutually exclusive?
+-------------------------------------------------------------------
+
+At present, this PEP explicitly allows, but does not formally recommend or
+require, build tools to back-fill the ``License`` core metadata field with
+the verbatim text from the ``License-Expression`` field. This would
+presumably improve backwards compatibility and was suggested
+by some on the Discourse thread. On the other hand, allowing it does
+increase complexity and is less of a clean, consistent separation,
+preventing the ``License`` field from being completely mutually exclusive
+with the new ``License-Expression`` field and requiring that their values
+match.
+
+As such, it would be very useful to have a more concrete and specific
+rationale and use cases for the back-filled data, and give fuller
+consideration to any potential benefits or drawbacks of this approach,
+in order to come to a final consensus on this matter that can be appropriately
+justified here.
+
+Therefore, is the status quo expressed here acceptable, allowing tools
+leeway to decide this for themselves? Should this PEP formally recommend,
+or even require, that tools back-fill this metadata (which would presumably
+be reversed once a breaking revision of the metadata spec is issued)?
+Or should this not be explicitly allowed, discouraged or even prohibited?
+
+
+Should custom license identifiers be allowed?
+---------------------------------------------
+
+The current version of this PEP retains the behavior of only specifying
+the use of SPDX-defined license identifiers, as well as the explicitly defined
+custom identifiers ``LicenseRef-Public-Domain`` and ``LicenseRef-Proprietary``
+to handle the two common cases where projects have a license, but it is not
+one that has a recognized SPDX license identifier.
+
+For maximum flexibility, custom ``LicenseRef-<CUSTOM-TEXT>`` license
+identifiers could be allowed, which could potentially be useful for niche
+cases or corporate environments where ``LicenseRef-Proprietary`` is not
+appropriate or insufficiently specific, but relying on mainstream Python
+build tooling and the ``License-Expression`` metadata field is still
+desirable to use for this purpose.
+
+This has the downsides, however, of not catching mispellings of the
```

Comment:
```suggestion
This has the downsides, however, of not catching misspellings of the
```

Reply:
Oh, the irony :laughing: Funny enough, when I was doing a quick proofread, I actually caught this same word mispelled [sic] somewhere else, and thought about how embarrassing it would be if that had made it through...heh. Fixed, thanks.